digits/frameworks/torch_framework.py

```diff
@@ -0,0 +1,162 @@
+# Copyright (c) 2015, NVIDIA CORPORATION.  All rights reserved.
+
+import os
+import digits
+import re
+import subprocess
+import time
+import tempfile
+import flask
+
+from framework import Framework
+from digits import utils
+from digits.config import config_value
+from digits.model.tasks import TorchTrainTask
+from digits.utils import subclass, override
+from errors import Error, NetworkVisualizationError, BadNetworkError
+
+@subclass
+class TorchFramework(Framework):
+
+    """
+    Defines required methods to interact with the Torch framework
+    """
+
+    # short descriptive name
+    NAME = 'Torch (experimental)'
```

Comment:
One bothersome thing about putting the "experimental" text in the `TorchFramework` class is that old torch jobs will continue to be flagged as "experimental" even when DIGITS upgrades to "full support" (whatever that means). Is there a way to put the "experimental" flagging in the template somewhere rather than here?